src/MooLite/plugins/WhisperManager/WhisperManagerPluginDisplay.vue

```diff
@@ -0,0 +1,216 @@
+
+    // Have to use ignore because it thinks chatMessageInput.value is type 'never'.
+    // @ts-ignore
+    chatMessageInput.value?.focus();
+};
+
+props.plugin.populateConversations();
+</script>
+
+<template>
+    <div class="flex flex-col space-y-1.5 h-full max-h-full mb-1.5">
+        <div class="flex flex-row justify-between items-center">
+            <h1
+                class="text-lg font-bold transition duration-700 ease-in-out"
+                :class="{
+                    'hover:text-cyan-300': contributorNames.includes(characterName),
+                    'bg-clip-text hover:text-transparent bg-gradient-to-br from-yellow-500 to-red-400 to-90% via-orange-300 via-30%':
+                        betaTesterNames.includes(characterName),
+                    'hover:text-fuchsia-300': helperNames.includes(characterName),
+                }"
+            >
+                Whisper Manager
+            </h1>
+            <button
+                v-if="isDev"
+                class="hover:scale-125 transition-transform aspect-square h-7 text-lg rounded-full text-gray-400 bg-gray-800 shadow-md hover:bg-gray-700"
+                @click="console.log('info', conversations)"
+            >
+                ℹ
+            </button>
+        </div>
+        <Accordion v-if="isDev">
+            <template #title>
+                <span>Settings</span>
+            </template>
+            <template #content>
+                <button
+                    @click="props.plugin.populateDevConversations()"
+                    class="text-sm bg-gray-800 mt-1.5 p-1.5 rounded-[4px]"
+                >
+                    Dev Data
+                </button>
+            </template>
+        </Accordion>
+        <div
+            v-if="!Object.keys(conversations).length"
+            class="flex flex-col justify-center gap-y-3 items-center w-full h-full text-center"
+        >
+            <span class="bold text-4xl">No whispers!</span>
+            <span class="text-xl">
+                Start one by clicking a users name in chat and selecting the whisper option. You can also type the
+                following in chat:
+            </span>
+            <code class="text-xl p-1.5 rounded-[4px] bg-gray-700">/w &lt;username&gt;</code>
+        </div>
+        <div v-else class="flex flex-col space-y-1.5 h-full max-h-full mb-1.5">
+            <button
+                v-if="Object.keys(conversations).filter((name) => conversations[name].hidden).length"
+                class="bg-gray-800 font-semibold p-1.5 rounded-[4px] text-gray-400 hover:text-gray-300 hover:bg-gray-700"
+                @click="unhideAllConversations()"
+            >
+                Unhide All
+            </button>
+            <input
+                v-if="Object.keys(conversations).filter((name) => !conversations[name].hidden).length > 4"
+                v-model="searchText"
+                type="text"
+                placeholder="Search tabs..."
```

Comment:
I would change this to something like "search users" to avoid giving the impression you can search form message content